### ISSUE TITLE
How to set the config file for multiple 2080ti (11G)?

### ISSUE THREAD

**Issue by ghost** [ldtho/PiFeNet#2]
I have 4 2080ti (11g), in order to get accurate results, can you tell me the detailed settings of step and steps_per_eval?
Thank you.

**Comment by ldtho** [comment]
Hi @o247541586, Sorry I couldn't be more helpful on this because I only trained and tested the model on a single GPU. Maybe refering back to the original repo https://github.com/traveller59/second.pytorch would be helpful.
Hope this helps

**Comment by o247541586** [comment]
> 



> Hi @o247541586, Sorry I couldn't be more helpful on this because I only trained and tested the model on a single GPU. Maybe refering back to the original repo https://github.com/traveller59/second.pytorch would be helpful. Hope this helps



> Hi @o247541586, Sorry I couldn't be more helpful on this because I only trained and tested the model on a single GPU. Maybe refering back to the original repo https://github.com/traveller59/second.pytorch would be helpful. Hope this helps

Thank you for your answer. Can you tell me why your baseline is so much higher than pointpillar?

**Comment by o247541586** [comment]
> Hi @o247541586, Sorry I couldn't be more helpful on this because I only trained and tested the model on a single GPU. Maybe refering back to the original repo https://github.com/traveller59/second.pytorch would be helpful. Hope this helps

Thank you for your answer. Can you tell me why your baseline is so much higher than pointpillar?

**Comment by ldtho** [comment]
The Pointpillars models (baselines) presented in the papers are reproduced by me, note that they are evaluated on the KITTI **val set**, not the test set. I'm just not sure that you're mentioning the performance on Val set or Test set, and which dataset?

**Comment by ldtho** [comment]
Hi, "Stacked Pillars" method sounds unfamiliar to me, can you give me the link to that method?
